distributions/R/scripts/new/header_parser.py

```diff
@@ -0,0 +1,171 @@
+import re
+import argparse
+import ctypes
+from sys import stderr
+
+from CppHeaderParser import CppHeader, CppMethod, CppVariable
+from ctype_translator import mapping
+from pathlib import Path
+import warnings
```

Comment:
For clarity sort imports according to https://www.python.org/dev/peps/pep-0008/#imports